brainscore/benchmarks/baker2022.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+
+import brainscore
+from brainscore.benchmarks import BenchmarkBase
+from brainscore.benchmarks.screen import place_on_screen
+from brainscore.metrics import Score
+from brainscore.metrics.accuracy_delta import AccuracyDelta
+from brainscore.model_interface import BrainModel
+from brainscore.utils import LazyLoad
+from brainscore.metrics.ceiling import SplitHalvesConsistencyBaker
+
+
+
+'''
+Made to directly address statement by Bowers et al. 2022:
+
+"Nevertheless, when DNNs classify objects based on shape, they use the wrong sort of shape representations.  
+For instance, in contrast with a large body of research showing that humans tend to rely on the global shape of 
+objects, Baker, Lu, Erlikhman, and Kellman (2018) showed that DNNs focus on local shape features.  That is, they 
+found that DNNs trained on ImageNet could correctly classify some silhouette images (where all diagnostic texture 
+information was removed), indicating that these images were identified based on shape.  However, when the 
+local shape features of the silhouettes were disrupted by including jittered contours, the models did much more poorly.
+By contrast, DNNs were more successful when the parts of the silhouettes were rearranged, a manipulation that 
+kept many local shape features but disrupted the overall shape.  Humans show the opposite pattern. "
+
+In other words:
+
+Local Disruption = Humans Ok, Models Bad
+Global Disruption = Humans Bad, Models Ok
+
+Baker 2022 tests for pictures that disrupt global features, but keep local features. That is, models should 
+do much more poorly on these images then humans. 
+
+'''
```

Comment:
this goes into a paper, not this benchmark definition imo (which will be around for longer)